src/main/java/uk/gov/justice/digital/delius/service/CustodyService.java

```diff
@@ -92,18 +96,21 @@ public Custody updateCustodyPrisonLocation(final String nomsNumber,
                 "toAgency", updateCustody.getNomsPrisonInstitutionCode());
 
         final var result = updateCustodyPrisonLocation(nomsNumber,
-                offender  -> getSingleActiveConvictionByOffenderIdAndPrisonBookingNumber(offender.getOffenderId(), bookingNumber),
+                offender  -> getAllActiveConvictionsByOffenderIdAndPrisonBookingNumber(offender.getOffenderId(), bookingNumber),
                 updateCustody.getNomsPrisonInstitutionCode());
         return result.map(success -> {
             switch (success.outcome) {
                 case Updated:
-                    telemetryClient.trackEvent("P2PTransferPrisonUpdated", telemetryProperties, null);
+                    telemetryClient.trackEvent("P2PTransferPrisonUpdated", add(telemetryProperties, "updatedCount", String.valueOf(success.custodyRecordsUpdated.size())), null);
                     break;
                 case NoUpdateRequired:
                     telemetryClient.trackEvent("P2PTransferPrisonUpdateIgnored", telemetryProperties, null);
                     break;
             }
-            return success.custody;
+            return success.custodyRecordsUpdated
+                .stream()
+                .max(Comparator.comparing(Custody::getSentenceStartDate))
```

Comment:
Just wondering why max?

Reply:
We only return one of the custody records, so max essentially is the latest one which is most appropriate.